app/images/apple_app_store_button/da.svg

```diff
@@ -0,0 +1,87 @@
+			c0.814,0,1.453-0.39,1.914-1.168C103.281,28.347,103.479,27.58,103.479,26.696z"/>
+		<path fill="#FFFFFF" d="M112.621,23.783c-0.211-0.039-0.436-0.059-0.672-0.059c-0.75,0-1.33,0.283-1.738,0.85
+			c-0.355,0.5-0.533,1.132-0.533,1.895v5.035h-2.131l0.02-6.574c0-1.106-0.027-2.113-0.08-3.021h1.857l0.078,1.836h0.059
+			c0.225-0.631,0.58-1.139,1.066-1.52c0.475-0.343,0.988-0.514,1.541-0.514c0.197,0,0.375,0.014,0.533,0.039V23.783z"/>
+		<path fill="#FFFFFF" d="M122.156,26.252c0,0.382-0.025,0.704-0.078,0.967h-6.396c0.025,0.948,0.334,1.673,0.928,2.173
+			c0.539,0.447,1.236,0.671,2.092,0.671c0.947,0,1.811-0.151,2.588-0.454l0.334,1.48c-0.908,0.396-1.98,0.593-3.217,0.593
+			c-1.488,0-2.656-0.438-3.506-1.313c-0.848-0.875-1.273-2.05-1.273-3.524c0-1.447,0.395-2.652,1.186-3.613
+			c0.828-1.026,1.947-1.539,3.355-1.539c1.383,0,2.43,0.513,3.141,1.539C121.873,24.047,122.156,25.055,122.156,26.252z
+			 M120.123,25.699c0.014-0.632-0.125-1.178-0.414-1.639c-0.369-0.593-0.936-0.889-1.699-0.889c-0.697,0-1.264,0.289-1.697,0.869
+			c-0.355,0.461-0.566,1.014-0.631,1.658H120.123z"/>
+	</g>
+	<g>
+		<g>
+			<path fill="#FFFFFF" d="M48.604,13.433h-1.048v-2.949h-2.832v2.949h-1.048V6.896h1.048v2.667h2.832V6.896h1.048V13.433z"/>
+			<path fill="#FFFFFF" d="M54.385,10.853c0,0.188-0.014,0.346-0.039,0.475h-3.143c0.013,0.466,0.164,0.821,0.455,1.067
+				c0.266,0.22,0.608,0.33,1.028,0.33c0.466,0,0.89-0.074,1.271-0.223l0.164,0.728c-0.446,0.194-0.973,0.291-1.581,0.291
+				c-0.73,0-1.305-0.215-1.722-0.645s-0.625-1.007-0.625-1.731c0-0.711,0.193-1.303,0.582-1.775
+				c0.407-0.504,0.956-0.756,1.648-0.756c0.679,0,1.193,0.252,1.542,0.756C54.246,9.77,54.385,10.265,54.385,10.853z M53.385,10.582
+				c0.007-0.311-0.062-0.579-0.203-0.805C53,9.485,52.723,9.34,52.348,9.34c-0.343,0-0.621,0.142-0.835,0.427
+				c-0.174,0.227-0.277,0.498-0.31,0.815H53.385z"/>
+			<path fill="#FFFFFF" d="M60,13.433h-1.048v-2.7c0-0.832-0.316-1.248-0.95-1.248c-0.311,0-0.562,0.114-0.757,0.343
+				c-0.193,0.229-0.291,0.499-0.291,0.808v2.796h-1.048v-3.366c0-0.414-0.013-0.863-0.038-1.349h0.921l0.049,0.737h0.029
+				c0.122-0.229,0.304-0.418,0.543-0.569c0.284-0.176,0.602-0.265,0.95-0.265c0.44,0,0.806,0.142,1.097,0.427
+				C59.82,9.398,60,9.918,60,10.61V13.433z"/>
+			<path fill="#FFFFFF" d="M64.221,9.504h-1.154v2.29c0,0.582,0.204,0.873,0.611,0.873c0.188,0,0.343-0.016,0.466-0.049l0.028,0.795
+				c-0.207,0.078-0.479,0.117-0.814,0.117c-0.414,0-0.737-0.126-0.97-0.378c-0.233-0.252-0.35-0.676-0.35-1.271V9.504H61.35V8.719
+				h0.688V7.855l1.028-0.31v1.173h1.154V9.504z"/>
+			<path fill="#FFFFFF" d="M69.672,7.312c0,0.188-0.062,0.339-0.184,0.456c-0.123,0.117-0.281,0.175-0.477,0.175
+				c-0.174,0-0.322-0.06-0.441-0.179c-0.12-0.12-0.18-0.27-0.18-0.451s0.062-0.33,0.185-0.446s0.274-0.175,0.456-0.175
+				c0.182,0,0.334,0.059,0.457,0.175C69.609,6.982,69.672,7.131,69.672,7.312z M69.555,13.433h-1.047V8.719h1.047V13.433z"/>
+		</g>
+	</g>
+</g>
+</svg>
```

Comment:
Because when [Apple provides you with all the localized badges](https://developer.apple.com/app-store/marketing/guidelines/#downloadOnAppstore), you don't try to re-invent the wheel.

Also, I don't want [lawyers](http://smartbusinessrevolution.com/wp-content/uploads/2010/10/blue_haired_lawyer_simpsons.jpg?w=150) busting down my door.

> Use only the artwork provided here. Never create your own App Store badge or alter the artwork in any way.

> Never translate App Store. Do not create your own translation or localized badge.

:grin: